docs/modules/ZSPySDK/examples/zs_api_sdk.py

```
@@ -212,12 +210,11 @@ def sync_volume_size(volume_uuid, session_uuid=None):
     action.uuid = volume_uuid
     try:
         evt = execute_action_with_session(action, session_uuid)
+        return evt.inventory
     except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)